client.go

```diff
@@ -53,13 +54,19 @@ func (c *client) Write(p []byte) (int, error) {
 	zap.L().Debug("write to client", zap.String("msg", string(p)))
 	n, err := c.conn.Write(p)
 	if err != nil {
-		zap.L().Error("write net failed", zap.String("addr", c.cliCtx.RemoteAddr),
-			zap.Int64("clientid", c.cliCtx.ID),
-			zap.String("namespace", c.cliCtx.Namespace),
-			zap.Bool("multi", c.cliCtx.Multi),
-			zap.Bool("watching", c.cliCtx.Txn != nil),
-			zap.String("command", c.cliCtx.LastCmd),
-			zap.String("error", err.Error()))
+		if err == io.EOF {
+			zap.L().Info("close connection", zap.String("addr", c.cliCtx.RemoteAddr),
+				zap.Int64("clientid", c.cliCtx.ID))
+			err = nil
```

Comment:
It is weird to set err to nil here. Maybe we can do this:

```
if err != nil {
  if err == io.EOF {
    log.Info...
  } else {
    log.Error...
    return 0, err
  }
}
return n, nil
```

Reply:
Ok， I've changed it as you said